lib/python/temporal/core.py

```diff
@@ -650,21 +667,28 @@ def init(raise_fatal_error=False, skip_db_version_check=False):
         if dbif.fetchone()[0]:
             db_exists = True
 
-    backup_howto = "The format of your actual temporal database is not " \
-                   "supported any more.\nSolution: You need to export it by " \
-                   "restoring the GRASS GIS version used for creating this DB"\
-                   ". From there, create a backup of your temporal database "\
-                   "to avoid the loss of your temporal data.\nNotes: Use " \
-                   "t.rast.export and t.vect.export to make a backup of your" \
-                   " existing space time datasets.To safe the timestamps of" \
-                   " your existing maps and space time datasets, use " \
-                   "t.rast.list, t.vect.list and t.rast3d.list. "\
-                   "You can register the existing time stamped maps easily if"\
-                   " you export columns=id,start_time,end_time into text "\
-                   "files and use t.register to register them again in new" \
-                   " created space time datasets (t.create). After the backup"\
-                   " remove the existing temporal database, a new one will be"\
-                   " created automatically.\n"
+    backup_howto = _(
+        "Please create a backup of your temporal database to avoid losing data.\n"
+    )
+    if tgis_db_version == 2:
+        backup_howto += _("Run t.downgrade command to downgrade your temporal database.\n")
```

Comment:
For the v2 case, it is not clear what backup means or when loosing data may occur. Do you expect users to do some backup before running t.downgrade? Maybe it can just say use t.downgrade or suggest export or coping mapset without listing the details. Or perhaps if you split the long instructions part can be backup and the other renewal. No strong opinion here.

Reply:
I'd keep it simple and only show the t.downgrade part for the v2 case. I've used both t.upgrade and downgrade and it's straightforward, no need to also do a backup (other than the regular grassdata backups).

Reply:
Personally I don't feel the sentence
```
Please create a backup of your temporal database to avoid losing data.
```
so much confusing. It suggests to do a backup of temporal database (by making copy of `sqlite.db` or dumping PG DB). But you are right, the sentence could be removed. I have not strong opinion here.

Reply:
It's not confusing but maybe not needed, i.e., is it risky to run t.downgrade? What about changing the order and start with: `Run t.downgrade...` and only afterwards something like: `Consider creating a backup of your temporal database to avoid losing data in case something goes wrong.` ?

Reply:
Changed in 6f74651. Message changed to:

```
ERROR: The format of your existing temporal database is not supported.
Temporal database version mismatch detected.
Run t.downgrade command to downgrade your temporal database.
Consider creating a backup of your temporal database to avoid losing data
in case something goes wrong.
Supported temporal database version is: 2
Your existing temporal database version: 3
Current temporal database info:
DBMI interface:..... sqlite3
Temporal database:..
/home/martin/grassdata/world_latlong_wgs84/tgis3/tgis/sqlite.db
```